src/plugins/blocksmith/components/vimeo-video-block-placeholder/index.jsx

```diff
@@ -1,15 +1,28 @@
 import React from 'react';
 import GenericBlockPlaceholder from '@triniti/cms/plugins/blocksmith/components/generic-block-placeholder';
+import Message from '@gdbots/pbj/Message';
+import PropTypes from 'prop-types';
 import PreviewComponent from './PreviewComponent';
 
-const config = {
-  icon: {
-    imgSrc: 'video',
-  },
-  label: 'Vimeo Video Block',
-  preview: {
-    component: PreviewComponent,
-  },
+const VimeoVideoBlockPlaceholder = (props) => {
+  const { block } = props;
+  const getUserName = block.getData().get('canvasBlock').get('user_name');
+  const fromUserName = getUserName ? `from ${getUserName}` : '';
```

Comment:
@tbone1020 these are kind of weird variable names. `getBlah` is generally used for functions. im thinking something like this...

```
const userName = block.getData().get('canvasBlock').get('user_name');

...some time later...

label: `Vimeo Video Block${userName ? ` from ${userName}` : ''}`,
```

Reply:
resolved